src/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<package xmlns="http://www.mendix.com/package/1.0/">
+    <clientModule name="Charts" version="1.0.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+        <widgetFiles>
+            <widgetFile path="BarChart/BarChart.xml"/>
+            <widgetFile path="LineChart/LineChart.xml"/>
+            <widgetFile path="PieChart/PieChart.xml"/>
+        </widgetFiles>
+        <files>
+            <file path="com/mendix/widget/custom/BarChart"/>
+            <file path="com/mendix/widget/custom/LineChart"/>
+            <file path="com/mendix/widget/custom/PieChart"/>
+        </files>
```

Comment:
Changing the packaga name to lower case as indicate above will have an effect here